### PR TITLE
Fix failing `test_batch_generation`  for bloom

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

#25571 changed this test. There was a tiny issue. See my comment along the change in this PR.

We should probably to change this variable names in multiple places for house keeping.

Also cc @gante  😄

### CODE REVIEW

File: tests/models/bloom/test_modeling_bloom.py [hunk 1]
```diff
@@ -449,9 +449,9 @@ def test_batch_generation(self):
 
         input_sentence = ["I enjoy walking with my cute dog", "I enjoy walking with my cute dog"]
 
-        input_ids = tokenizer.batch_encode_plus(input_sentence, return_tensors="pt", padding=True)
```
Comment (ydshieh) [inline tests/models/bloom/test_modeling_bloom.py:452]: not actually ids

File: tests/models/bloom/test_modeling_bloom.py [hunk 2]
```diff
@@ -449,9 +449,9 @@ def test_batch_generation(self):
...
-        input_ids = input_ids["input_ids"].to(torch_device)
```
Comment (ydshieh) [inline tests/models/bloom/test_modeling_bloom.py:453]: no ids

File: tests/models/bloom/test_modeling_bloom.py [hunk 3]
```diff
@@ -449,9 +449,9 @@ def test_batch_generation(self):
...
-        attention_mask = input_ids["attention_mask"]
```
Comment (ydshieh) [inline tests/models/bloom/test_modeling_bloom.py:454]: no `attention_mask` in the actual `ids`

File: tests/models/bloom/test_modeling_bloom.py [hunk 4]
```diff
@@ -449,9 +449,9 @@ def test_batch_generation(self):
...
+        inputs = tokenizer.batch_encode_plus(input_sentence, return_tensors="pt", padding=True)
```
Comment (ydshieh) [inline tests/models/bloom/test_modeling_bloom.py:452]: good name and avoid failure.

**Review by ArthurZucker — APPROVED** [review]
😅 Nice catch !

**Comment by ydshieh** [comment]
> 😅 Nice catch !

I was caught by the failed CI 😢

**Pull request merged**